### DISCUSSION TITLE
Integración con proveedor de firma electrónica (Signio)

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#36] [General]
Integración que buscará ubicar dentro de SincoSGD los documentos firmados electrónicamente desde un aplicativo externo

**Comment by NestorFGG** [comment]
Se encuentra en proceso de pruebas de integración

**Comment by NestorFGG** [comment]
Se evalúan cambios con el tercero debido a su administración interna de accesos, y estamos a la espera de sus ajustes.

**Comment by aleja550** [comment]
La integración se tiene que replantear para realizarse con Estampilla.
